internal/resource/config/license/license_data_source.go

```diff
@@ -58,7 +58,7 @@ type licenseDataSourceModel struct {
 // GetSchema defines the schema for the datasource.
 func (r *licenseDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
 	schemaDef := schema.Schema{
-		Description: "Describes a License.",
+		Description: "Describes a license summary object.",
```

Comment:
I think you could remove the "object" here

Reply:
I actually went back and forth on including that word.